frontend/resources/public/css/app.css

```diff
@@ -859,3 +859,62 @@ input.leaflet-control-layers-selector[type="radio"] {
     cursor: default;
 }
 /* L.Control.Zoominfo.css end */
+
+.floating-pills {
+    position: absolute;
+    z-index: 2000;
+    top: -40px;
+    left: 10px;
+    transition: top .5s, left .5s;
+    white-space: nowrap;
+}
+
+@media (min-width:768px) {
+	.floating-pills {
+        top: 10px;
+	}
+}
+
+@media (min-width:768px) and (max-width:991px) {
+	.floating-pills {
+		left: calc(305px + 20px);
```

Comment:
My only question is where these numbers are coming from; is that 305px etc from the current sidebar?

Reply:
Yes, that's right

Reply:
305px (etc.) is the sidebar width, and 20px is the 10px of "padding" either side of the sidebar.

Reply:
Cool; something to keep in mind I guess (actually, is that what the sidebar does to the rest of the floating controls?  I assume so)..